### ISSUE TITLE
VB compiler will not report issues about code inside a lambda if that lambda does not bind.

### ISSUE THREAD

**Issue by CyrusNajmabadi** [dotnet/roslyn#53238]
For example:

```vb
Imports System.Linq
Class C
    Sub New()
        Dim s As Action = Sub()
                              Dim a = New C(0)
```

Note the lack of an `Imports System` here.  Withtou that, 'Action' is unknown, which means we don't give any any diagnostics (hidden or otherwise) *at all* within the lambda.  This is problematic as it means general IDE features driven off of diagnostics do not work.

**Comment by Youssef1313** [comment]
Is that a dupe of https://github.com/dotnet/roslyn/issues/32628?

**Comment by CyrusNajmabadi** [comment]
Possibly.  But that doesn't seem to have much info in it.

**Comment by jaredpar** [comment]
They look like dupes to me but I agree the other doesn't have much info in it. Duped that one against this one.
